packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -435,9 +437,19 @@ export default class OperatorModeStackItem extends Component<Signature> {
     this.containerEl = el;
   };
 
+  private closeItem = async () => {
+    this.isClosing = true;
+    // wait for the animation to complete
+    await timeout(100);
+    this.args.close(this.args.item);
+  };
+
   <template>
     <div
-      class='item {{if this.isBuried "buried"}}'
+      class='item
+        {{if this.isBuried "buried"}}
+        {{if this.isLastItem "open"}}
+        {{if this.isClosing "dismissed"}}'
```

Comment:
Maybe the class could have a more descriptive name, for example  "closing-animation"

Reply:
Indeed the namings are much clearer now, thanks for the suggestion @jurgenwerk